test/nightly/deploy.test.ts

```diff
@@ -67,11 +72,16 @@ suite('Create Web App and deploy', async function (this: ISuiteCallbackContext):
 
 // The workspace folder that vscode is opened against for tests
 async function getWorkspacePath(testWorkspaceName: string): Promise<string> {
+    let workspacePath: string = '';
     const workspaceFolders: vscode.WorkspaceFolder[] | undefined = vscode.workspace.workspaceFolders;
     if (!workspaceFolders || workspaceFolders.length === 0) {
         throw new Error("No workspace is open");
     } else {
-        const workspacePath: string = workspaceFolders[0].uri.fsPath;
+        for (const obj of workspaceFolders) {
```

Comment:
Technically having multi-root workspaces is a different test case, though this is a limitation I'm not sure what to do about.  @EricJizbaMSFT any ideas?

Reply:
I see what you mean, but I think this is the easiest way to get the test done so I'm fine with it. This actually isn't unique to this PR. It was already testing a multi-root workspace since it was using a `.code-workspace` file (even though that workspace file only happened to have one folder)